libraries/aws/shadow/cbmc/sources/README.md

```diff
@@ -0,0 +1 @@
+../aws-templates-for-cbmc-proofs/template-for-repository/sources/README.md
```

Comment:
I believe that this is also symlinked

Reply:
please review [this commit](https://github.com/aws/aws-iot-device-sdk-embedded-C/pull/1134/commits/87cb11ae1496f618c50d49ad2ed7fd5c3630a3ec) for the update.